eng/versioning/external_dependencies.txt

```diff
@@ -278,6 +281,15 @@ resourcemanager_com.jcraft:jsch;0.1.55
 # sdk\resourcemanager\azure-resourcemanager-compute\pom.xml
 resourcemanager_com.microsoft.azure:azure-storage;6.1.0
 
+# sdk\resourcemanager\azure-resourcemanager-samples\pom.xml
+resourcemanager_org.apache.httpcomponents:httpclient;4.5.12
+
+# sdk\resourcemanager\azure-resourcemanager\pom.xml
+# sdk\resourcemanager\azure-resourcemanager-network\pom.xml
+# sdk\resourcemanager\azure-resourcemanager-resources\pom.xml
+# sdk/resourcemanager/azure-resourcemanager-storage/pom.xml
+# sdk\resourcemanager\azure-resourcemanager-samples\pom.xml
+resourcemanager_com.google.guava:guava;28.1-jre
```

Comment:
Reason why we need this 2 exceptions? I assume these are not version sensitive libs, and we seem only use them in a few samples.

Reply:
For httpclient, it would cause error if we use lower version. (No test with specific version, but error raised when use httpclient that inside the dependency)
For guava, it could be change to lower version. Although I think it is not a better choice.

Reply:
guava has been reverted.

Reply:
What kind of error, and which code is using it? I cannot find the code that using the lib.

Reply:
You can remove this dependency and run test, it will raise a class not found error. (Inside dependency).